app/models/item.rb

```diff
@@ -19,6 +19,10 @@ def self.search_by_params(param)
 		if param[:name] && (param[:min_price] || param[:max_price])
 			raise ArgumentError.new("Cannot search by both name and price")
 		end
+		
+		if param[:min_price].to_f < 0 || param[:max_price].to_f < 0
+			raise ArgumentError.new("price parameters cannot be less than 0")
+		end
```

Comment:
Nice use of conditionals and raise argument error, this taught me something new.